.gitignore

```diff
@@ -69,6 +69,7 @@ website/mfrcache/
 website/mfrtemp/
 website/osfstoragecache/
 website/gpg/
+website/static/sitemaps/
```

Comment:
Just a drive-by comment: It might make sense to generate the sitemaps to `website/static/built/sitemaps`. The `/built/` directory is used for generated assets and is already gitignored.